### PR TITLE
(VDB-1199) Add extractDiffs to docker-compose

### PR DESCRIPTION
**Pull request by rmulhol**
- Also remove unnecessary env data from other images

### CODE REVIEW

File: dockerfiles/docker-compose.yml [hunk 1]
```diff
@@ -38,14 +35,25 @@ services:
     command: ["./wait-for-it.sh", "db:5432", "--strict", "--", "./startup_script.sh"]
     restart: unless-stopped
 
+  extract-diffs:
```
Comment (elizabethengelman) [inline dockerfiles/docker-compose.yml]: 🎉

File: dockerfiles/docker-compose.yml [hunk 2]
```diff
@@ -38,14 +35,25 @@ services:
...
+      - DATABASE_PORT=5432
+      - DATABASE_USER=vdb
+      - DATABASE_PASSWORD=vdb
+    depends_on:
+      - db
+    command: ["./wait-for-it.sh", "db:5432", "--strict", "--", "./startup_script.sh"]
+
   execute:
     image: makerdao/vdb-execute:latest # (needs wait-for-it.sh)
-    env_file: mainnet-1.0.1.env
```
Comment (elizabethengelman) [inline dockerfiles/docker-compose.yml]: I think I'm missing something - how are we getting the contract addresses, etc in without this env file now?

Comment (rmulhol) [reply dockerfiles/docker-compose.yml]: Moved to [config](https://github.com/makerdao/vdb-mcd-transformers/blob/staging/environments/docker.toml#L479) - apparently the actual deployment setup requires some cumbersome detail for env vars, so moving toward minimizing the details that need to remain in the env

**Comment by rmulhol** [comment]
@elizabethengelman any blockers to getting your approval for this one?

**Comment by elizabethengelman** [comment]
@rmulhol whoops, so sorry, meant to approve earlier!

**Pull request merged**